tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/model/NewLiferayPluginProjectOp.java

```diff
@@ -17,28 +17,28 @@
 import com.liferay.ide.core.ILiferayProjectProvider;
```

Comment:
This class should be changing very little for this new module project wizard.  I see that you have renamed many classses.  Lets avoid that unnecessary class renaming, please restore the original names.  In order to avoid similiar names put all of the new classes in a new package called "com.liferay.ide.project.core.modules"